README.md

```diff
@@ -254,3 +254,7 @@ Note: The [Patchwork](https://github.com/FlorentRevest/vscode-patchwork) and
 [Git Send Email](https://github.com/FlorentRevest/vscode-git-send-email)
 extensions are homegrown and maintained in other git repositories.
 Pull-requests or bug reports are also welcome there.
+
+## LPC Talk
+[SLIDES](https://lpc.events/event/17/contributions/1614/attachments/1210/2474/VSCode%20for%20kernel%20development.pdf)
+[![VSCode for kernel development](https://img.youtube.com/vi/nmYaSqe0fGg/0.jpg)](https://www.youtube.com/watch?v=nmYaSqe0fGg)
```

Comment:
Please add an empty line between the above line and this one or the markdown renderer puts both on one line and this looks odd.